imports/server/configureLogger.ts

```diff
@@ -26,8 +28,15 @@ logger.format = format.combine(
     level, label, message, [userIdSymbol]: userId, error,
     ...rest
   }) => {
+    const ctx: string[] = [];
+    if (workersCount > 1) {
```

Comment:
I wouldn't be sad about including this regardless

Reply:
Eh. It feels like it's just noisy in local dev